docs/README.md

```diff
@@ -123,3 +130,420 @@ something like
+`CreatePost` command, we authorized `all` to execute it. This time we will be doing the
+same for the read model. Furthermore, We will exclude the `content` field from the `Post`
+entity so it won't be visible when querying this read-model. To do so, edit the class to
+look like this:
+
+```typescript
+// src/read-models/PostReadModel.ts
+@ReadModel({
+  authorize: 'all'// Specify authorized roles here. Use 'all' to authorize anyone
+})
+export class PostReadModel {
+  public constructor(
+    public id: UUID,
+    readonly title: string,
+    readonly author: string,
+  ) {}
+
+  @Projects(Post, "postId")
+  public static projectPost(entity: Post, currentPostReadModel?: PostReadModel): PostReadModel {
+      return new PostReadModel(entity.id, entity.title, entity.author);
+  }
+
+}
+```
+
+#### 6. Deployment
+
+We have everything we need to deploy our application to the cloud. It is as simple as
+running this command:
+
+```bash
+boost deploy -e production
+```
+
+It will take a couple of minutes to deploy all the resources. Once finished, you will see
+information about your stack endpoints. For this example, we will only need to pick the
+API endpoint. Look into the output for something like `backend-application-stack.baseRESTURL`
+e.g.
+
+```text
+https://<some random number>.execute-api.us-east-1.amazonaws.com/production/graphql
+```
+
+We are close to testing our app, and to do so, we will use the GraphQL API endpoint.
+Let's address that in the next section.
+
+#### 7. Testing
+
+Let's get started testing the project. We will perform three actions:
+
+- Add a couple of posts
+- Retrieve all posts
+- Retrieve a specific post
+
+Booster applications provide you with a GraphQL API out of the box. Commands are mapped
+to mutations and read models to queries. To perform calls to the GraphQL API, you can use
+any HTTP client you want; we recommend you to use [Postwoman](https://postwoman.io/graphql).
+
+##### 7.1 Creating posts
+
+Use your favorite GraphQL client to run these mutations. No authorization header is
+required since we have allowed `all` to execute our commands and query the read models.
+
+```graphql
+mutation {
+  CreatePost(
+    input: {
+      postId: "95ddb544-4a60-439f-a0e4-c57e806f2f6e"
+      title: "Build a blog in 10 minutes with Booster"
+      content: "I am so excited to write my first post"
+      author: "Boosted developer"
+    }
+  )
+}
+```
+
+```graphql
+mutation {
+  CreatePost(
+    input: {
+      postId: "05670e55-fd31-490e-b585-3a0096db0412"
+      title: "Booster framework rocks"
+      content: "I am so excited for writting the second post"
+      author: "Another boosted developer"
+    }
+  )
+}
+```
+
+The expected response for each of those requests should be:
+
+```json
+{
+  "data": {
+    "CreatePost": true
+  }
+}
+```
+
+Note:
+
+> The IDs are generated on the client-side. For this example, we are not validating ID
```

Comment:
I don't know why but the first time that I read the sentence, it seems to me that there is also the possibility to configure the server-side ID generation

Reply:
You are right @rdiaz82, thank you for pointing this out. I'll rephrase this to make the point clearer

Reply:
what do you think about this:

> With Booster, IDs are generated on the client-side. When running production applications
> consider adding validation for ID uniqueness. In this example, we have used [a UUID generator](https://www.uuidgenerator.net/version4)

Reply:
Well, this will change soon, we've planned to return the `entityID`s of all events emitted, instead of just that unmeaningful "true"